src/data-validation/schema/input-dataset-info.schema.json

```diff
@@ -89,11 +89,8 @@
         "title",
         "version",
         "name",
-        "image",
         "description",
         "featuresDataPath",
-        "viewerSettingsPath",
```

Comment:
So, I'm not sure if I've found a bug or if I'm misunderstanding what the intended behavior is here.

I created a dataset, and then I've gone into `dataset.json` and `feature_defs.json` and set anything that is no longer required to an empty string. Everything seems to work except for `viewerSettingsPath` which if I set that to an empty string and then `npm run validate-datasets` I get an error:
```
[Error: EISDIR: illegal operation on a directory, read] {
  errno: -21,
  code: 'EISDIR',
  syscall: 'read'
}
```
Can you help me understand if I'm taking the right approach in reviewing this code?

Reply:
You're on the right track by testing this through either removing the key or setting it to an empty string! However, you shouldn't be able to create a dataset, because this maintenance branch doesn't have the code for dataset creation. 

You could try this:
Instead of creating a dataset on your own, edit any `dataset.json` or `feature_defs.json` within the existing datasets(e.g. `data/dataset-1-4-1`), and run `npm run validate-datasets` to see if the validation passes.

Feel free to DM me if you encounter any further issues. Thank you for taking a look at the pr!